data/json/recipes/recipe_food.json

```diff
@@ -9100,6 +9100,36 @@
     ]
   ]
 },{
+   "type" : "recipe",
+   "result": "quesadilla_cheese",
+   "category": "CC_FOOD",
+   "subcategory": "CSC_FOOD_SNACK",
+   "skill_used": "cooking",
+   "difficulty": 0,
+   "time": 5000,
+   "reversible": false,
+   "autolearn": true,
+   "qualities" : [
+     {"id":"COOK","level":1,"amount":1}
+  ], "tools": [
+   [
+      [ "hotplate", 3 ],
+      [ "char_smoker", 1 ],
+      [ "toolset", 3 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "cheese", 1 ],
+      [ "can_cheese", 1],
+      [ "cheese_hard", 1]
+    ],
+    [
+      [ "tortilla_corn", 2 ]
```

Comment:
Ya kinda like the tacos have no flatbread. It starts getting complicated with all the different materials. Maybe we will have a different system in the future that get the materials for things from the ingredients and not have them be defined before hand. Til then I guess I'll leave it out.